charts/README.md

```diff
@@ -0,0 +1,159 @@
+# puris
```

Comment:
We need to adjust the overall chart structure. I had a mistake in mind:
charts
charts/puris
charts/puris/charts/backend
charts/puris/charts/frontend